recipes/datma/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "datma" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+  
+source:
+  url: https://github.com/andvides/DATMA/releases/download/v1.0/codes_v1.zip
+  sha256: 0ba0797434d7318c96f514bcf2fc141c886ad4156d69ac4ae0058d6c9dbfcfbc
+
+build:
+  number: 0
+  noarch: python
```

Comment:
Please change this to `noarch: generic`, since it's not a standard python package (just some scripts).